patches/busybox/1.36.1/busybox-1.36.1-kernel-6.8.patch

```diff
@@ -0,0 +1,53 @@
+http://lists.busybox.net/pipermail/busybox/2024-March/090678.html
+https://bugs.gentoo.org/926872
+
+Linux v6.8-rc1 removed the definitions related to CBQ making tc fail to
+build. Add some #ifdefs to handle this missing support.
+--- a/networking/tc.c
++++ b/networking/tc.c
+@@ -231,6 +231,13 @@ static int cbq_parse_opt(int argc, char **argv, struct nlmsghdr *n)
+ 	return 0;
+ }
+ #endif
++
++#ifndef TCA_CBQ_MAX
++/*
++ * Linux v6.8-rc1~131^2~60^2^2 removed the uapi definitions for CBQ.
```

Comment:
It looks like this line is corrupted.
```suggestion
+ * Linux v6.8-rc1 removed the uapi definitions for CBQ.
```

Reply:
Yeah you're right, I didn't really try to fix all that from the patch in the ML. Done.